src/mlpack/tests/wgan_test.cpp

```diff
@@ -268,7 +268,7 @@ BOOST_AUTO_TEST_CASE(WGANGPMNISTTest)
     arma::mat samples;
     noise.imbue( [&]() { return noiseFunction(); } );
 
-    generator.Forward(noise, samples);
+    wgan_gp.Generator().Forward(noise, samples);
```

Comment:
Please use camel casing for all names. See https://github.com/mlpack/mlpack/wiki/DesignGuidelines#naming-conventions for all information.

Reply:
I think it is done exactly in the way specified? I don't see an issue here.

Reply:
Instead of ``wgan_gp`` we should use ``wganGP``or something similar.